### DISCUSSION TITLE
Relay is forward or redirect?

### DISCUSSION THREAD

**Discussion by Worldwidebrine** [PrismarineJS/bedrock-protocol#298] [Q&A] [answered]
I think it's forwarding.
Would you add redirecting?
`/transferserver` has been removed.
I want to transfer players from my cheap VPS(can't handle forwarding) to my local machine(dynamic IP and dynamic port due to NAT).

**Comment by extremeheat** [comment]
I don't understand this question. Please show your code and expected outcome.

**Comment by Worldwidebrine** [comment]
Custom servers can transfer a player to another server(different address, different port).
I want to do that but I haven't found any useful API.
Did I miss something?

**Comment by Worldwidebrine** [comment]
I want to
```js
const { Redirect } = require('bedrock-protocol')
const redirect = new Redirect({
  version: '1.16.220',
  host: '0.0.0.0',
  port: 19132,
  destination: {
    host: "8.8.8.8",
    port: 19132
  }
})
redirect.listen()
```
Players connect to this server will be transferred to 8.8.8.8:19132
It's not heavy because players will connect to 8.8.8.8:19132 directly

**Comment by extremeheat** [comment, marked as answer]
You need to use the Server to implement something like this. When a client joins, you need to send the appropriate packet (a transfer packet) to allow them to join the server. bedrock-protocol provides a proxy under the "Relay" object (documented), but this creates a non-transparent proxy.

Closing issue and moving to discussion as this is not an issue with bedrock protocol.
